.github/workflows/dotnet.yml

```diff
@@ -0,0 +1,25 @@
+name: .NET
```

Comment:
This repo is currently using Azure Pipelines for automatic builds & testing. While we can consider switching to GH Actions in the future, please remove this for now, thanks.